### PR TITLE
🎨 [Frontend] TIP v3 Feedback

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Feedback from 20.06 meeting:
- Login page
  - Add Powered by Sim4Life logo

image

- Request Account
  - Company Name -> Organization
  - Remove Application
  - Remove Description

image

- New Plans:
  - Show Credits icon instead of plus button for billable plans
  - When starting a billable show credits warning if within threshold

Warning

- Credits Indicator:
  - Bring it back, but this time just the Credits Icon. Its color depends on the amount credits. When clicked, show the number of credits and provide access to the Billing Center

Good
Indicator

Not so Good
Indicator_red

## Related issue/s

related to https://github.com/ITISFoundation/osparc-issues/issues/1309

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by elisabettai — APPROVED** [review]
Very nice! Quite smart the way of handling credits. 😉 

Minor: did the POs suggest the icon for credits? It doesn't remind me of credits, rather it looks like a disk/storage

**Comment by odeimaiz** [comment]
> Minor: did the POs suggest the icon for credits? It doesn't remind me of credits, rather it looks like a disk/storage

They suggested a coin, coins, pile of coins....

Feel free to pick from here (make sure it was released before 5.0.5):
https://fontawesome.com/v5/search?o=r&m=free&s=solid

**Review by sanderegg — APPROVED** [review]

**Comment by elisabettai** [comment]
> > Minor: did the POs suggest the icon for credits? It doesn't remind me of credits, rather it looks like a disk/storage
> 
> They suggested a coin, coins, pile of coins....
> 
> Feel free to pick from here (make sure it was released before 5.0.5): https://fontawesome.com/v5/search?o=r&m=free&s=solid

Mmh, didn't find anything much better.
If we can use pro, I'd go for https://fontawesome.com/v5/icons/coin?f=classic&s=regular
If not, maybe: https://fontawesome.com/v5/icons/coins?f=classic&s=solid (which is not much different from what you already have, though...)

**Comment by odeimaiz** [comment]
> > > Minor: did the POs suggest the icon for credits? It doesn't remind me of credits, rather it looks like a disk/storage
> > 
> > 
> > They suggested a coin, coins, pile of coins....
> > Feel free to pick from here (make sure it was released before 5.0.5): https://fontawesome.com/v5/search?o=r&m=free&s=solid
> 
> Mmh, didn't find anything much better. If we can use pro, I'd go for https://fontawesome.com/v5/icons/coin?f=classic&s=regular If not, maybe: https://fontawesome.com/v5/icons/coins?f=classic&s=solid (which is not much different from what you already have, though...)

No pro... and "coins", which was my first choice, was released in 5.0.13. We would have to upgraded, it might take some time....

**Review by pcrespov — APPROVED** [review]

**Review by GitHK — APPROVED** [review]
🪙

File: services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    const buttonSize = 26;
+    const billingCenterButton = new qx.ui.form.Button().set({
+      appearance: "form-button-outlined",
+      width: buttonSize,
+      height: buttonSize,
+      alignX: "center",
+      alignY: "middle",
+      center: true,
+      icon: "@FontAwesome5Solid/ellipsis-h/12"
```
Comment (jsaq007) [inline services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js:51]: From my understanding the 3 dot or ellipses should remain consistent. Either horizontal or vertical but not both... We should remain the same for the button so I would suggest changing this to vertical too

File: services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+  construct: function() {
+    this.base(arguments);
+
+    this._setLayout(new qx.ui.layout.Grow());
+
+    this.set({
+      zIndex: osparc.utils.Utils.FLOATING_Z_INDEX,
+      maxWidth: this.self().WIDTH,
+      backgroundColor: "background-main-2",
```
Comment (jsaq007) [inline services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js:29]: I think if we are going to use these we need to define the use-case for each as this name means very little which is why I starting to move more and more away. I think also the color should also be the same as our other dropdown menu for consistency?

File: services/static-webserver/client/source/class/osparc/desktop/credits/Utils.js [hunk 3]
```diff
@@ -20,12 +20,24 @@ qx.Class.define("osparc.desktop.credits.Utils", {
 
   statics: {
     DANGER_ZONE: 25, // one hour consumption
+    CREDITS_ICON: "@FontAwesome5Solid/database/",
```
Comment (jsaq007) [inline services/static-webserver/client/source/class/osparc/desktop/credits/Utils.js:23]: Not such a big fan of database icon in this context but just my 2c worth

**Review by jsaq007 — CHANGES_REQUESTED** [review]
Just the couple issues I thought could be addressed.

File: services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js:51]: the actions are different, but ok

File: services/static-webserver/client/source/class/osparc/desktop/credits/Utils.js (hunk 3 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/credits/Utils.js:23]: Just read the conversation above and what I commented during the daily

**Comment by odeimaiz** [comment]
@jsaq007 

CreditsAppearance

File: services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js (hunk 2 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js]: Implemented 👍

**Review by jsaq007 — APPROVED** [review]
Nice, thanks

**Pull request merged**